smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/TypeScriptWriter.java

```diff
@@ -225,10 +225,10 @@ boolean writeShapeDocs(Shape shape, UnaryOperator<String> preprocessor) {
                     if (shape.getTrait(DeprecatedTrait.class).isPresent()) {
                         DeprecatedTrait deprecatedTrait = shape.expectTrait(DeprecatedTrait.class);
                         String deprecationMessage = deprecatedTrait.getMessage()
-                            .map(msg -> " " + msg)
-                            .orElse("");
-                        String deprecationString = "@deprecated" + deprecationMessage;
-                        docs = docs + "\n\n" + deprecationString;
+                            .map(msg -> " " + msg) 
```

Comment:
no need to prefix empty space here since one already exists after the `@deprecated` tag.